tool/grind.dart

```diff
@@ -21,19 +21,19 @@ void analyze(GrinderContext context) {
 
 @Task()
 @Depends(init)
-void tests(GrinderContext context) {
+void tests() {
   Tests.runCliTests(context);
 }
 
 @Task()
 @Depends(init)
-Future testsWeb(GrinderContext context) {
+Future testsWeb() {
   return Tests.runWebTests(context, directory: 'web', htmlFile: 'web.html');
 }
 
 @Task()
 @Depends(init)
-Future testsBuildWeb(GrinderContext context) {
+Future testsBuildWeb() {
   return Pub.buildAsync(context, directories: ['web']).then((_) {
```

Comment:
I guess we can't make `GrinderContext` params from methods which already have named arguments optional since dart doesn't allow a method to have both named and positional optionals.  I guess we can't provide a non-breaking upgrade path for that.  I think maybe those methods are relatively new though, and thus wouldn't have that many users yet, so should we ahead and remove the params now?